### PR TITLE
Current order

### PR DESCRIPTION
**Pull request by domisobi**
This isn't fully functioning, but I think we'll end up adding stripe here so I'd like to merge it sooner rather than later!

### CODE REVIEW

File: client/components/orders/CurrentOrder.jsx [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+        setOrder(order)
+        setLoading(false)
+        console.log(order)
+
+        const {data} = await axios.get(`/api/user/order/${order.id}`, 
+            { headers: { authorization: token } }
+            )
+            const drinks = data.orderDrinks 
+            setDrinks(drinks)
+            console.log(drinks)
```
Comment (justinmattos) [inline client/components/orders/CurrentOrder.jsx:29]: Does the `order` you get from `/api/user/order/current` not include the drinks? I think it should, but if not then it's not working properly.

File: client/components/orders/CurrentOrder.jsx [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+          drink.quantity--
+          newSubtotal  = subtotal - drink.drink.price
+        }
+      }
+      return drink
+    }).filter(drink => drink.quantity !== 0)
+    // update drink here with put route?
+    setDrinks(newDrinks)
+    setSubTotal(newSubtotal)
+  }
```
Comment (justinmattos) [inline client/components/orders/CurrentOrder.jsx:75]: I think we'll also want these to send PUT requests to the server to modify the order in our database, right?

File: public/bundle.js.LICENSE.txt [hunk 3]
```diff
@@ -0,0 +1,50 @@
+/*
+object-assign
+(c) Sindre Sorhus
+@license MIT
+*/
+
```
Comment (justinmattos) [inline public/bundle.js.LICENSE.txt]: I don't think we want to commit this file

File: server/api/user/order.js [hunk 4]
```diff
@@ -27,7 +27,7 @@ router.get('/all', requireUserToken, async (req, res, next) => {
 router.get('/current', requireUserToken, async (req, res, next) => {
   try {
     const { user } = req;
-    const order = await user.currentOrder();
+    const order = await user.currentOrder(); // grab subtotal too?
```
Comment (justinmattos) [inline server/api/user/order.js:30]: Yeah, good idea. There's an instance method, so you could add this below before sending to the client
```javascript
order.subtotal = await order.getSubtotal();
```

File: client/components/orders/CurrentOrder.jsx (hunk 1 above)
Comment (domisobi) [reply client/components/orders/CurrentOrder.jsx:29]: I wasn't getting this to include drinks, but I think it makes sense to have to grab the order first to make sure we're in the same order and not someone's past order.

File: client/components/orders/CurrentOrder.jsx (hunk 2 above)
Comment (domisobi) [reply client/components/orders/CurrentOrder.jsx:75]: definitely, I believe @arjanmitra is working on these

**Pull request merged**